utils/futures.ts

```diff
@@ -0,0 +1,18 @@
+import { NetworkId, NetworkIdByName } from '@synthetixio/contracts-interface';
+
+import futuresMarketsKovan from 'synthetix/publish/deployed/kovan-ovm/futures-markets.json';
+import futuresMarketsMainnet from 'synthetix/publish/deployed/mainnet-ovm/futures-markets.json';
+
+export const getMarketKey = (asset: string | null, networkId: NetworkId) => {
+	if (networkId === NetworkIdByName['mainnet-ovm']) {
+		return futuresMarketsMainnet.find((market) => market.asset === asset)?.marketKey || 'sETH';
+	} else if (networkId === NetworkIdByName['kovan-ovm']) {
+		return futuresMarketsKovan.find((market) => market.asset === asset)?.marketKey || 'sETH';
+	} else {
+		return 'sETH';
+	}
+};
```

Comment:
I chose to default to 'sETH', since thats the new default market. However, we never actually call this function with a falsy input, so it's basically unreachable code put there to satisfy TS.

Reply:
this sounds reasonable to me, I can wait for this to merge and rebase to incorporate this into the open prs I have dealing with asset vs marketKey